packages/trader/src/App/Containers/trade-header-extensions.tsx

```diff
@@ -23,11 +22,9 @@ const TradeHeaderExtensions = observer(({ store }: TradeHeaderExtensionsProps) =
 
     const populateHeaderfunction = React.useCallback(() => {
         const header_items = show_component ? (
-            <MobileWrapper>
-                <TraderProviders store={store}>
-                    <PopulateHeader />
-                </TraderProviders>
-            </MobileWrapper>
+            <TraderProviders store={store}>
```

Comment:
Do we need to add `isMobile` check here? From hooks

Reply:
Also, in this files there is a usage of `isMobile` from `@shared`. Is this expected?

Reply:
I will update it, it was causing infinite re-render